.github/workflows/nodejs.yaml

```diff
@@ -18,4 +18,8 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - run: npm ci
       - run: npm test
+      - run: npm run predeploy
+      - run: aws s3 cp lambda.zip s3://covid-vaccine-scraper-test/lambda.zip
+      - run: aws lambda update-function-code --function-name scrapeCovidAppointmentDataTEST \
```

Comment:
@j256 As mentioned for line 22, AWS credentials will be necessary to run this command against the platform.